declarative_authorization.gemspec

```diff
@@ -4,14 +4,14 @@ Gem::Specification.new do |s|
   s.name = "declarative_authorization"
   s.version = "1.0.0.pre"
 
-  s.required_ruby_version = ">= 1.8.6"
+  s.required_ruby_version = ">= 2.0.0"
   s.authors = ["Steffen Bartsch"]
   s.summary = "declarative_authorization is a Rails plugin for maintainable authorization based on readable authorization rules."
   s.email = "sbartsch@tzi.org"
   s.files = %w{CHANGELOG MIT-LICENSE README.rdoc Rakefile authorization_rules.dist.rb garlic_example.rb init.rb} + Dir["app/**/*.rb"] + Dir["app/**/*.erb"] + Dir["config/*"] + Dir["lib/*.rb"] + Dir["lib/**/*.rb"] + Dir["lib/tasks/*"] + Dir["test/*"]
   s.has_rdoc = true
   s.extra_rdoc_files = ['README.rdoc', 'CHANGELOG']
   s.homepage = %q{http://github.com/stffn/declarative_authorization}
-
-  #s.add_dependency('rails', '>= 2.1.0')
+  s.add_dependency('ruby_parser', '~> 3.6.6')
+  s.add_dependency('rails', '>= 4.0.0', '<= 5.0.0')
```

Comment:
Shouldn't this be more like `'< 5.0.0'` or `'< 5.1'`? (i.e. "less than", not "less or equal") 
The current version of this line allows Rails 5.0.0, but not Rails 5.0.1, which IMHO doesn't make much sense (Correct me if I'm wrong)